data/avgi/avgi 1 first contact.txt

```diff
@@ -0,0 +1,1000 @@
+			`	He looks at his hands. "So I... I made the decision, as captain. I volunteered my crew to investigate it, to travel through that hyperlane in the name of science. For what the Deep holds highest, because I guess I wanted... I don't know. Not the prestige. But to live up to what the Deep's supposed to be, the ones at the forefront of human exploration and research. They took me in when I had nowhere else to go, and I guess I wanted to repay that somehow. And now, now a dozen good men and women are dead, and the rest are stuck here behind a hyperlane that closed the moment we went through it, living among aliens fighting a war of extinction that they probably can't win."`
+			choice
+				`	"The other members of your crew, where are they now?"`
+
+			`	"There are a few of us left," he sighs, "but, well, I'm not on the best of terms with them. They blame me for bringing them here, and I can't fault them for it."`
+			choice
+				`	"That seems unfair. There's no way you could have known what would happen."`
+					goto blame
+				`	"Is there a chance of reconciling with them?"`
+			`	"I've made... a lot of mistakes. Ones with a price paid in lives. But," he looks down, "...maybe. They don't hate me or anything like that, we just... keep our distance. Sora's off doing who knows what with the Twilight Guard on Weledos, and Leon is on and off with those mining guilds to the East. There are a few others left too, but like I said. A price paid in lives."`
+				goto end
+
+			label blame
+				`	"I wish it was that simple. I made more mistakes after that, ones with a price measured in lives. Eleven dead after our ship was disabled, all because I ordered us to get too close to the scary-looking alien ships in the hope we could communicate with them. Sure, you could say I couldn't have known the Aberrant would try to kill us, but I ignored the advice of my crew, just because I was desperate to get home. Sora, Leon, they both told me we should run, but I didn't listen to them."`
+
+			label end
+			`	You sit together in silence for a few more minutes, when Darius seems to receive a message on his communicator. "I get patched in to the emergency alert system," he explains as he reads it. Suddenly, he pales. "A scout is reporting a massive invasion headed towards the Ensemble system. Dozens of Aberrations, flying straight through the Shroud." He looks up at you. "They don't normally try to go too deep in the Shroud. Something about it spooks them, I think, but Ensemble is shallow enough that they've attacked it before. Nothing of this scale though, and... damn, Sora's on Weledos, the moon in the Ensemble system."`
+			`	He looks at you, hopeful. "We've got to do something. The Avgi are assembling a relief force around the Navigeo Yards, their big shipyard not too far from here. Come on, let's join it."`
+				accept
+
+
+mission "Avgi: Defend Ensemble 2"
```

Comment:
I'd also have a larger look at this mission. You have to travel quite a ways to the battle, and it is very likely that the common random spawns or unfortunate positioning when jumping in will lead to you, or the rest of the fleet following you, getting nuked.
Personally, I had to do around 8 runs just to make it to the system with most of the fleet intact, and 13 to actually survive the battle. (This was using a Bactrian with 3 olofez'.) I may have just been unlucky many of those times, but the setup of the mission (having a bunch of nuke happy ships travel a long distance with you) leads to a high likelyhood of something going wrong in a manner that leads to your or their deaths.

I think reducing the travel distance would be the single most impactful change to make this mission less tedious to play, other than that I might look at the fleet composition (the glacial turn speed of the "Atenuatia (Atomic)" variant contributed the most to this issue) or possibly reduce the range of the missiles slightly.

Reply:
Maybe switch the Atomic variant's powerful thrusters for powerful steering?

Reply:
seems like the steering was switched out for the nukes. I don't think downgrading the engine would be particularly good because then it would have either power or thrust problems (depending on how you outfit it.) Sacrificing the cargo would probably get you there.

Reply:
I just realized some things
1. that wouldn't work because the Ophrys stockpiles take up basically all the cargo space.
2. All it needs to have enough space for the larger steering is one less Ophrys stockpile, it has a little less endurance with a full compliment fo nuclear fighters, but is still very functional. (also by the time the stocks run out in testing I am usually only left with one or two fighters/drones operational (because the rest nuked themselves.)